### PR TITLE
Fix broken function.

### PR DESCRIPTION
**Pull request by Xuanwo**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -81,5 +63,5 @@ exports.ext = function (filename) {
 exports.url = function (path, ssl) {
   ssl = ssl || false;
 
-  return (ssl ? '//dn-staticfile.qbox.me' : 'http://cdn.staticfile.org') + path;
+  return (ssl ? 'https://cdn.staticfile.org' : 'http://cdn.staticfile.org') + path;
```
Comment (ikbear) [inline lib/index.js]: 现在域名都统一起来了，这里面格式直接保留 `//cdn.staticfile.org` 就可以了。

**Comment by ikbear** [comment]
有界面修改的贴一个截图，对比前后改进效果。

cli 截 terminal 结果。

File: lib/index.js (hunk 1 above)
Comment (Xuanwo) [reply lib/index.js]: 好的

**Comment by Xuanwo** [comment]
Before this patch, sfile is broken. And the patch‘s target is to fix broken function, I removed some feature and find a better way to use this tool.

Firstly, we should search keyword, this keyword don't need to be accurate:

screenshot from 2016-11-10 16-53-30

After that, we get files:

screenshot from 2016-11-10 16-50-49

**Comment by Xuanwo** [comment]
@ikbear Please review again.

**Pull request closed without merging**